examples/next/streetlights.yml

```diff
@@ -55,7 +55,7 @@ channels:
     parameters:
       streetlightId:
         $ref: '#/components/parameters/streetlightId'
-    publish:
+    subscribe:
```

Comment:
These are `commands` to turn on, turn off and dim the lights ya?  In messages below this it says "Command a particular streetlight to turn the lights on or off." So I Believe these should be `publish`, not `subscribe`, since the client would be publishing those commands to the service. Maybe these were already the new way (or the 1.0 way?).

Reply:
There's a mistake in the description of the message. It's because, in the beginning, this example was representing a specific streetlight API. Right now it's representing a system that can deal with streetlights. Like a control panel or something similar.

Thanks for pointing out. I'll make sure every description gets nailed in this document.

Reply:
Just realized it's correct. This example is describing a "control panel" so `subscribe` is correct.

Reply:
See the description of the API:

The Smartylighting Streetlights API allows you to remotely manage the city lights.

### Check out its awesome features:

  * Turn a specific streetlight on/off 🌃
  * Dim a specific streetlight 😎
  * Receive real-time information about environmental lighting conditions 📈